src/prettytable/prettytable.py

```diff
@@ -518,6 +518,9 @@ def _validate_attributes(self, name, val):
     ##############################
     # ATTRIBUTE MANAGEMENT       #
     ##############################
+    @property
+    def rows(self):
+        return self._rows
```

Comment:
I think my earlier review got lost, take 2:

Can we return a copy here so the original is not writable?

```suggestion
        return self._rows[:]
```